src/modules/rdf/functions.cpp

```diff
@@ -517,7 +527,7 @@ bool RDFModule::sumUnweightedGR(GenericList &processingData, const ProcessPool &
 
     // Calculate overall density of combined system
     double rho0 = std::accumulate(configWeights.begin(), configWeights.end(), 0.0, [totalWeight](double acc, auto pair) {
-        return acc + pair.second / totalWeight / pair.first->atomicDensity();
+        return acc + pair.second / totalWeight / pair.first->atomicDensity().value();
```

Comment:
Error check, but this is going to be a bit trickier, since it's inside the lambda.  My initial suggestion would be
```suggestion
        if (!acc) return {};
        auto density = pair.first->atomicDensity{};
        if (!density) return {};
        return *acc + pair.second / totalWeight / *density;
```

And then return an error message if the overall accumulate returns null.

Reply:
Rather than "complicate" the `accumulate` call here, I've elected to perform a direct check in the preceding loop so that no lambda-based check is necessary.